### PR TITLE
Some changes for TDT4195

### PR DESCRIPTION
**Pull request by bartvbl**
I found that the glfw error callback was a little more useful than only reporting that a window could not be initialised when creating the OpenGL context. The other two commits are a couple of settings I think should be altered.

### CODE REVIEW

**Comment by bartvbl** [comment]
I think we should also consider removing the camera.hpp class for the TDT4195 course, as I think the focus should be there on manipulating transformation matrices manually. As this is not really the main focus of the labs in TDT4230, we can include it there to save some time.

File: gloom/src/main.cpp [hunk 1]
```diff
@@ -9,6 +9,11 @@
 // Standard headers
 #include <cstdlib>
 
+// A callback which allows GLFW to report errors whenever they occur.
+static void glfwErrorCallback(int error, const char *description)
+{
+    fprintf(stderr, "GLFW returned an error:\n\t%s (%i)\n", description, error);
+}
```
Comment (aleksaro) [inline gloom/src/main.cpp:16]: Add another blank space above and below the function. Also, remove punctuation after "_occur_" in the comment.

File: gloom/src/main.cpp [hunk 2]
```diff
@@ -25,6 +30,8 @@ GLFWwindow* initialise()
     glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
     glfwWindowHint(GLFW_OPENGL_FORWARD_COMPAT, GL_TRUE);
 
+    glfwSetErrorCallback(glfwErrorCallback);
```
Comment (aleksaro) [inline gloom/src/main.cpp:33]: Move error callback below the _glfwWindowHint_ groups and write a short comment above it.

File: gloom/src/program.cpp [hunk 3]
```diff
@@ -13,6 +13,8 @@ void runProgram(GLFWwindow* mWindow)
     glEnable(GL_DEPTH_TEST);
     glDepthFunc(GL_LESS);
 
+    glEnable(GL_CULL_FACE);
```
Comment (aleksaro) [inline gloom/src/program.cpp:16]: Remove blank space above to group it with the rest of the culling setup.

**Pull request closed without merging**